packages/ocean-lending/src/clients/integration.ts

```diff
@@ -0,0 +1,87 @@
+import Web3 from "web3";
+import { Account } from "web3-core";
+import { ERC20Token } from "./erc20Token";
+import { OToken, OTokenParameter } from "./oToken";
+import { Comptroller } from "./comptroller";
+import { JumpInterestV2 } from "./jumpInterestV2";
+import { TxnOptions } from "options";
+
+export class IntegrationClient {
+  // The account to use for operations
+  private account: Account;
+  // The comptroller to use for operations
+  private comptroller: Comptroller;
+  // The jumpInterestV2 to use for operations
+  private jumpInterestV2: JumpInterestV2;
+  // The confirmations to use for operations
+  private confirmations: TxnOptions;
+
+  constructor(
+    account: Account,
+    comptroller: Comptroller,
+    jumpInterestV2: JumpInterestV2,
+    confirmations: TxnOptions
+  ) {
+    this.account = account;
+    this.comptroller = comptroller;
+    this.jumpInterestV2 = jumpInterestV2;
+    this.confirmations = confirmations;
+  }
+
+  /**
+   * @param oToken The oToken client object
+   * @param blockTime The number of seconds per block
+   */
+  public async poolInfo(oToken: OToken, blockTime: number): Promise<any> {
+    const [liquidity, borrowRateAPY, supplyRateAPY] = await Promise.all([
+      this.comptroller.getAccountLiquidity(this.account.address),
+      this.jumpInterestV2.getBorrowRateAPY(oToken, blockTime),
+      this.jumpInterestV2.getSupplyRateAPY(oToken, blockTime),
+    ]);
+
+    return {
+      address: oToken.address,
+      liquidity: liquidity,
+      borrowAPY: borrowRateAPY,
+      depositAPY: supplyRateAPY,
+    };
+  }
+
+  public async collateralSettings(): Promise<any> {
+    const [collateralFactor, closeFactor, liquidationIncentive] =
+      await Promise.all([
+        this.comptroller.collateralFactor(this.account.address),
+        this.comptroller.closeFactor(this.account.address),
+        this.comptroller.liquidationIncentive(),
+      ]);
+
+    const canBeCollateral = collateralFactor != 0;
+
+    return {
+      canBeCollateral: canBeCollateral,
```

Comment:
Here can just have {
 canBeCollateral,
...
}
no need repeating, strange, the linter never inform?